File: Fibonacci_Sequence/Fibonacci_Sequence.js
const Fibonacci_Sequence = (n) => {
    const fib = [ 0, 1 ];
    for (i = 2; i < n; i++) {
        fib[ i ] = fib[ i - 1 ] + fib[ i - 2 ];
        // fib.push(fib[ i - 1 ] + fib[ i - 2 ]);
    }
    return fib;
};

console.log(Fibonacci_Sequence(7));

// Big-O = O(n)

File: Algorithm/Fibonacci_Sequence/Fibonacci_Sequence.py
def Fibonacci_Sequence(n):
    fib = [0, 1]
    for i in range(2,n):
        fib.append(fib[i - 1] + fib[i - 2])

    return fib

print(Fibonacci_Sequence(7))

# fib = [0, 1]
# for i in range(2,5):
#     print(f"hellow {i}")
# fib.append(2)

# print(fib)

